### ISSUE TITLE
Increase raid loot drops for 10 man [#SomeChanges Suggestion]

### ISSUE THREAD

**Issue by ChristianBlom84** [ClassicWoWCommunity/cata-classic-bugs#1168]
### Description
With the faster content cadence, gearing in 10 man raids is going to be harder than ever. Loot disparity was a big part of the discrepancy between 10 and 25 man raiding in original Cataclysm, and it's going to be even worse now.

2/3 pieces of loot for non-tier/tier bosses for 10 man and 6/9 for 25 man means triple the loot for 25 man groups, while also wasting much less of it.

Please consider increasing the 10 man numbers to 3/4 or 3/5 and if needed increase the 25 man drops - they might be strapped for loot as well with the increased pace.

**Comment by GeorgiosP** [comment]
There needs to be an incentive for 25 man raids otherwise no one would do them if the loot ratio is better optimized for 10 man 😢

**Comment by Kenniflopper** [comment]
Yeah but the shorter seasons makes it so almost no 10 man will be close to full gear once the next raids are released, this is a good idea.

**Comment by GeorgiosP** [comment]
well, I think thats true for both raid sizes correct? considering class distribution and going for BIS isn't possible for all your raiders unless you are split raiding. 

example there is only 1 caster ring with haste mastery 😖  

This should include both raid sizes if we just want full bis in three month time lines. 🤷🏼

**Comment by ChristianBlom84** [comment]
Well, right now a normal ICC 25 boss drops what, 3 pieces of loot? Maybe 4? It's already almost doubled for 25 man groups in Cata. If you want to go with the old argument that you need excessive amounts of loot to make 25 man viable, then give 10 man 3/4 or 3/5 pieces and just triple that for 25 man.

**Comment by AidanZMoon** [comment]
A boss like Nefarion drops 12 pieces of loot on 25H. 6 drops, 3 H tier, 3 normal tier.
Is that still not enough?

**Comment by GeorgiosP** [comment]
I think if its a tier token issue for 10 man its probably reasonable to up those drops similar to how the number of tokens were increased in ICC

**Comment by GeorgiosP** [comment]
But a net increase in ALL loot should be done for both if thats the intended goal

**Comment by ChristianBlom84** [comment]
I think loot needs to increase across the board since they're accelerating the content pace so much.

**Comment by jlldarocha** [comment]
Since every tier will last half of what we had on original cataclysm, its fair to say thats a needed change.

I think a 1 item on  all bosses along with a extra generic token on tier bosses would solve the disenchanting all the loot/tiers going into OS because we got vanquisher shoulders 5 weeks in a row for 10s. 25s might be good enough with 2 extra items and 2 tokens, because usually you dont waste as many items as you do on 10s because there's more variety on classes/specs than on 10s.

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
